src/ipc-handlers.ts

```diff
@@ -1051,7 +1051,8 @@ END` );
 			: {};
 
 		if ( preferredTerminal === ( 'warp' as SupportedTerminal ) ) {
-			return promiseExec( `start "" "warp" "--cwd=${ targetPath }"`, {
+			const encodedPath = encodeURIComponent( targetPath );
+			return promiseExec( `start "" "warp://action/new_tab?path=${ encodedPath }"`, {
```

Comment:
I chose to open a new tab instead of a new window, as the user might already have windows open.